realm/realm-library/build.gradle

```diff
@@ -37,6 +37,7 @@ repositories {
 
 dependencies {
     provided 'io.reactivex:rxjava:1.1.0'
+    provided 'junit:junit:4.12'
```

Comment:
hmm, I'm feeling wrong to put `RealmRobolectricRule` into main library.

What do you think? @realm/java